experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -472,11 +473,23 @@ export const FormAudience = ({
           <Form.Group as={Col} controlId="minVersion">
             <Form.Label className="d-flex align-items-center">
               Min Version
+              {experiment.isWeb && (
+                <>
+                  <Info
+                    data-tip={TOOLTIP_DISABLED_FOR_WEBAPP}
+                    data-testid="tooltip-disabled-min-version"
+                    width="20"
+                    height="20"
+                    className="ml-1"
+                  />
+                  <ReactTooltip />
+                </>
+              )}
             </Form.Label>
             <Form.Control
               {...formControlAttrs("firefoxMinVersion")}
               as="select"
-              disabled={isLocked!}
+              disabled={isLocked! || experiment.isWeb}
```

Comment:
could we add this to the `isLocked` definition? 

```
const isLocked = (isLiveRollout && !getStatus(experiment).draft) || experiment.isWeb;
```

Reply:
or are there fields that we want locked using `isLocked` that don't line up with locked for web?

Reply:
yes there are certain fields which are not consistent with `isLocked`

Reply:
:(